script/packtpub.py

```diff
@@ -30,6 +33,12 @@ def __init_headers(self):
 
     def __log_response(self, response, method='GET', detail=False):
         print '[-] {0} {1} | {2}'.format(method, response.url, response.status_code)
+        if self.__debug:
```

Comment:
@windhamwong this will append a different log on file, correct? While I would like to find a prettier solution to redirect the same output on console or file or both. What do you think?